packages/manager-components/src/components/table/table.spec.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { OsdsText } from '@ovhcloud/ods-components/text/react';
+import { ODS_TEXT_SIZE } from '@ovhcloud/ods-components/text';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { Table, TableProps } from './table.component';
+import { mockColumns, mocksData } from './table.stories';
+import { render, waitFor, within } from '../../utils/test.provider';
+
+const setupSpecTest = async (
+  customProps?: Partial<TableProps>,
+  table?: TableProps,
+) =>
+  waitFor(() =>
```

Comment:
it doesn't look like 'waitFor' is needed here, is it ?

Reply:
Good, it not necessary I will update it in the others components